src/CMakeLists.txt

```diff
@@ -333,6 +333,23 @@ IF((APPLE) AND (PKG_CONFIG_FOUND))
   ENDIF()
 ENDIF()
 
+IF((APPLE) AND (PKG_CONFIG_FOUND))
+  PKG_SEARCH_MODULE(PC_HUNSPELL hunspell libhunspell)
+  IF(PC_HUNSPELL_FOUND)
+    IF(PC_HUNSPELL_hunspell_FOUND)
+      MESSAGE(STATUS "Using pkg_config, found \"hunspell\" version: ${PC_HUNSPELL_hunspell_VERSION} with:")
+    ELSEIF(PC_HUNSPELL_libhunspell_FOUND)
+      MESSAGE(STATUS "Using pkg_config, found \"libzzip\" version: ${PC_HUNSPELL_libhunspell_VERSION} with:")
```

Comment:
Woops - forgot to edit that copied text - should of course read "... found \"libhunspell\" version..." but it is only a reporting text and the script works... :blush: